selfdrive/car/hyundai/carcontroller.py

```diff
@@ -96,8 +96,7 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
 
 
     # 20 Hz LFA MFA message
-    if frame % 5 == 0 and self.car_fingerprint == CAR.SONATA:
+    if frame % 5 == 0 and (self.car_fingerprint == CAR.SONATA or self.car_fingerprint == CAR.PALISADE) :
```

Comment:
can you change this to:
```suggestion
    if frame % 5 == 0 and self.car_fingerprint in [CAR.SONATA, CAR.PALISADE]:
```

That makes it cleaner to all cars